templates/sample-project/README.md

```
@@ -0,0 +1,64 @@
+   ```bash
+   npm start
+   ```
+
+4. **Try AI Features**
+   - Open `src/api/hello.ts` in VibeCode
+   - Start typing and watch AI code completion suggestions
+   - Try asking the AI to add new features
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)